spec/controllers/article_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe ArticlesController do
+  describe 'GET index' do
+    let(:user) { FactoryGirl.create(:user) }
+    before(:all) { 30.times { FactoryGirl.create(:user) } }
+    after(:all)  { User.delete_all }
```

Comment:
Esto no haría falta. Si tenés bien configurado rspec, deberia después de cada `it` borrar toda la base de datos y cargarla de vuelta. Esto es para garantizar que no quede la base de datos sucia entre its